docs/specs/localization.yml

```diff
@@ -745,9 +745,6 @@ repos:
           localization:
             mapping: Branch
             bilingual: true
-          gitHub:
-            userCache: github-user.json
```

Comment:
don't we need userCache config anymore?

Reply:
there is no such config now, it is a known location in cache folder.